### PR TITLE
feature(v4.0): refactoring

### LINKED ISSUE #78: Use a settings file to declare the architectures compatible with RAPL
Currently, the architectures compatible with RAPL are directly written inside the Helper. Given the fact the architectures are constantly evolving, such parameters should be put inside a configuration file instead.

### PR DESCRIPTION
**Pull request by mcolmant**
Each base component has now is own supervisor, a Clock is not started automatically anymore (allows to have external "ticks", a new display for InfluxDB was written.

and, Closes #78

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/Spirals-Team/powerapi/pull/80" target="_blank">ReviewNinja

File: powerapi-core/src/main/scala/org/powerapi/PowerMeter.scala [hunk 1]
```diff
@@ -23,248 +23,227 @@
...
     new PowerMeter(system, modules)
   }
 }
 
 /**
- * A PowerModule groups a sets of tightly coupled elements that need to be deployed together.
- *
- * @author <a href="mailto:romain.rouvoy@univ-lille1.fr">Romain Rouvoy</a>
- */
+  * A PowerModule groups a sets of tightly coupled API components that need to be deployed together.
```
Comment (rouvoy) [inline powerapi-core/src/main/scala/org/powerapi/PowerMeter.scala]: Replace "a sets" by "a set" !fix

File: powerapi-core/src/main/scala/org/powerapi/core/ClockActors.scala [hunk 2]
```diff
@@ -88,11 +90,11 @@ class ClockChild(eventBus: MessageBus, frequency: FiniteDuration) extends ActorC
 }
 
 /**
- * This clock listens the bus on a given topic and reacts on the received message.
- * It is responsible to handle a pool of clocks for the monitored frequencies.
- *
- * @author <a href="mailto:maxime.colmant@gmail.com">Maxime Colmant</a>
- */
+  * This clock listens the bus on a given topic and reacts on the received message.
```
Comment (rouvoy) [inline powerapi-core/src/main/scala/org/powerapi/core/ClockActors.scala]: Replace "listen" by "listens to "

Comment (mcolmant) [inline powerapi-core/src/main/scala/org/powerapi/core/ClockActors.scala]: !fixed

File: powerapi-core/src/main/scala/org/powerapi/PowerMeter.scala (hunk 1 above)
Comment (mcolmant) [inline powerapi-core/src/main/scala/org/powerapi/PowerMeter.scala]: !fixed

**Pull request merged**